### PR TITLE
teacher tool - run prettier

### PR DESCRIPTION
**Pull request by eanders-ms**
I know everyone hates prettier. I don't like it either. But I do think style consistency is important. I'm open to other solutions.

I'm hoping we will remember to run prettier as part of each PR.

There is one thing be alert for: calls to `lf(...)` cannot be broken into multiple lines. Our gulp script cannot handle this currently and will error out. TODO to fix this.

### CODE REVIEW

File: teachertool/src/state/actions.ts [hunk 1]
```diff
@@ -62,24 +62,12 @@ type HideModal = ActionBase & {
...
-    | RemoveNotification
-    | SetProjectMetadata
-    | SetEvalResult
-    | SetTargetConfig
-    | SetCatalog
-    | SetSelectedCriteria
-    | RemoveCriteriaInstance
-    | ShowModal
-    | HideModal;
+export type Action = PostNotification | RemoveNotification | SetProjectMetadata | SetEvalResult | SetTargetConfig | SetCatalog | SetSelectedCriteria | RemoveCriteriaInstance | ShowModal | HideModal;
```
Comment (thsparks) [inline teachertool/src/state/actions.ts:65]: Could we `// prettier-ignore` both of the export statements in this file? I realize these might fit within our expanded max line length, but I feel like it's okay to make an exception here...

Comment (eanders-ms) [reply teachertool/src/state/actions.ts:65]: Certainly. There is a tension with increased line width vs what we'd prefer to have stacked. We may want to bring the max line width in a bit. 200 is very wide.

Comment (eanders-ms) [reply teachertool/src/state/actions.ts]: Setting width to 120 corrected this. And the results seemed more reasonable to me in general.

Comment (thsparks) [reply teachertool/src/state/actions.ts]: Seems like you found the sweet spot!

**Review by thsparks — APPROVED** [review]

**Pull request merged**